Automattic/jetpack
# VideoPress: show pricing based on coupon discount

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: show pricing based on coupon discount

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Disconnect the site
* Go to the VideoPress dashboard
* Confirm you see the discount in the pricing section
* Confirm the app leads to the checkout page by clicking on the "Get VideoPress" button

<img width="440" alt="image" src="https://user-images.githubusercontent.com/77539/203141951-bdf8c114-0069-4672-b2b6-9cada1995dde.png">
